### ISSUE TITLE
Twilio\Http\Response::__toString() causes array to string conversion E_NOTICE

### ISSUE THREAD

**Issue by rtek** [twilio/twilio-php#387]
```
    public function getContent() {
        return json_decode($this->content, true);
    }

    public function __toString() {
        return '[Response] HTTP ' . $this->getStatusCode() . ' ' . $this->getContent();
    }
```

getContent() typically returns array - in this case using the raw json would be the fix.

**Comment by ihumanable** [comment]
Thank you for writing in, seems like a simple fix, will address.

**Comment by efossier** [comment]
Fixed by 4587453153d8e7b90c881b75496b7d5cd941bc13
